test/library/standard/Path/Prachi-Muskaan/testcases_expandUser.chpl

```diff
@@ -0,0 +1,10 @@
+/* Test cases to validate expandUser function in Path.chpl */
+
+use Path;
+
+writeln(expandUser('~user/'));
+writeln(expandUser('~USER/'));
+writeln(expandUser('~/'));
+writeln(expandUser('user/'));
+writeln(expandUser('~'));
```

Comment:
I don't have a problem with these last three test cases and am pleased you thought of them :)

Reply:
It also might be good to double check the behavior when `~` is later in the path, e.g. `/path/gets/to/here/~/whoops`